docker/Dockerfile.caddy-watcher

```
@@ -1,7 +1,26 @@
+FROM debian:bookworm-slim
+
+ARG NODE_VERSION=25.8.2
+ARG TARGETARCH
 
 ARG PNPM_VERSION=10.31.0
 
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends ca-certificates curl libatomic1 xz-utils \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN set -eux; \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)